symtab_builder.cc

```diff
@@ -55,6 +56,18 @@ bool SymtabBuilder::Resolve(const std::string& name, const std::string& soname,
     } else {
         auto found = src_syms_.find({name, soname, version});
 
+        if (found == src_syms_.end() && soname.empty() && version.empty()) {
+            // When we cannot find a symbol using (name, soname, version), we
+            // try to find a symbol using only a name. In this case, you must
+            // not forget to check VERSYM_HIDDEN. Although there is no
+            // description of VERSYM_HIDDEN in glibc, you can find it in
+            // binutils source code.
+            // https://github.com/gittup/binutils/blob/8db2e9c8d085222ac7b57272ee263733ae193565/include/elf/common.h#L816
```

Comment:
Better to move this link (and the comment after "Although") to util.h?

Reply:
OK.